developers/weaviate/search/bm25.md

```diff
@@ -21,8 +21,8 @@ This page shows you how to perform keyword searches.
 
 A keyword search looks for objects that contain the search terms in their properties according to the selected [tokenization](../config-refs/schema.md#property-tokenization). The results are scored according to the [BM25F](https://en.wikipedia.org/wiki/Okapi_BM25) function, and thus a keyword search is also called the `bm25` search in Weaviate. It is also called a sparse vector search.
 
-:::info Related pages
-- [API References: Search operators # BM25](../api/graphql/search-operators.md#bm25)
+:::info Search Accuracy
+Starting in `v1.19.0`, BM25 search has improved accuracy for schemas with a large number of properties and for zero-length properties. If you are using an earlier version, please upgrade.
 :::
 
```

Comment:
I would prefer to revert this note to keep the "related pages" info at the top. And then let's merge the PR.

Reply:
Moved to bottom of page as per discussion